app/client/components/pages/uploadPage.jsx

```diff
@@ -7,69 +7,29 @@ import UploadedTable from "../uploader/uploadedTable.jsx";
 
 const mapStateToProps = state => {
   return {
-    loggedIn: state.login.token != null,
     spreadsheetData: state.spreadsheetUpload.data
   };
 };
 
-const mapDispatchToProps = dispatch => {
-  return {};
-};
-
-const uploadPage = ({ loggedIn, spreadsheetData, location }) => {
+const uploadPage = ({ spreadsheetData, location }) => {
   const is_vendor = location.search.indexOf("type=vendor") !== -1;
 
   if (spreadsheetData) {
-    var inner_div = (
-      <UploadedTable data={spreadsheetData} is_vendor={is_vendor} />
-    );
+    return <UploadedTable data={spreadsheetData} is_vendor={is_vendor} />;
```

Comment:
don't approve unless is_vendor works (or rather, the user can specify the user type)

Reply:
@tristanhcole Is this reviewable yet?

Reply:
got vendor specification working by just allowing an `account_type` column